Package sources are insufficient to build a binary package

[logs](https://github.com/amirreza8002/django-valkey/actions/runs/11358630302/job/31593446993#step:6:119)

hi it seems libvalky can't build on 3.13
i don't care much but it should be added to setup.py  for people to know

This is a bit of a drive-by report because it's not blocking me but i figured i'd share in case it's any use.

I encountered this too on MacOS 15.0.1

```
$ uname -a
Darwin r2 24.0.0 Darwin Kernel Version 24.0.0: Tue Sep 24 23:39:07 PDT 2024; root:xnu-11215.1.12~1/RELEASE_ARM64_T6000 arm64
```

I was using python 3.13.0 with uv 0.4.26, both from homebrew:

```
$ uv add valkey[libvalkey]
...
error: Failed to prepare distributions
  Caused by: Failed to download and build `libvalkey==4.0.0`
  Caused by: Build backend failed to build wheel through `build_wheel` (exit status: 1)

[stdout]
running bdist_wheel
running build
running build_py
creating build/lib.macosx-15.0-arm64-cpython-313/libvalkey
copying libvalkey/version.py -> build/lib.macosx-15.0-arm64-cpython-313/libvalkey
copying libvalkey/__init__.py -> build/lib.macosx-15.0-arm64-cpython-313/libvalkey
copying libvalkey/libvalkey.pyi -> build/lib.macosx-15.0-arm64-cpython-313/libvalkey
copying libvalkey/py.typed -> build/lib.macosx-15.0-arm64-cpython-313/libvalkey
running build_ext
building 'libvalkey.libvalkey' extension
creating build/temp.macosx-15.0-arm64-cpython-313/src
creating build/temp.macosx-15.0-arm64-cpython-313/vendor/libvalkey/src
clang -fno-strict-overflow -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -O3 -Wall -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX15.sdk -Ivendor/libvalkey/include -Ivendor/libvalkey/include/valkey -I/Users/craig/.cache/uv/builds-v0/.tmpBZcPHr/include -I/opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13 -c src/libvalkey.c -o build/temp.macosx-15.0-arm64-cpython-313/src/libvalkey.o -std=c99

[stderr]
In file included from src/libvalkey.c:1:
src/libvalkey.h:5:10: fatal error: 'valkey/read.h' file not found
    5 | #include <valkey/read.h>
      |          ^~~~~~~~~~~~~~~
1 error generated.
error: command '/usr/bin/clang' failed with exit code 1
  Caused by: This error likely indicates that you need to install a library that provides "valkey/read.h" for libvalkey@4.0.0
```

The same `uv add valkey[libvalkey]` command in a python 3.12 project just succeeds installing libvalkey (doesn't try to build from source).

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -3,6 +3,7 @@ include MANIFEST.in
 include README.md
 include src/*.[ch]
 include vendor/libvalkey/COPYING
+include vendor/libvalkey/include/valkey/*.h
 include vendor/libvalkey/src/*.[ch]
 graft tests
 global-exclude __pycache__

